### PR TITLE
[WIP]Travel class

### PR DESCRIPTION
**Pull request by prochetab1999**

### CODE REVIEW

File: src/components/TravelClass.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+const TravelClassDetails = ({ travelClassName, availability }) => (
+  <div className="travelClass-leftGrid">
+    <div className="travelClass-name">{travelClassName}</div>
+    <div className="travelClass-availability">{availability}</div>
+  </div>
+
+);
+const TravelClassPrice = ({ fareDetails }) => (
+  <div className="travelClass-rightGrid">
```
Comment (hariharanweb) [inline src/components/TravelClass.js:12]: can we use price component

Comment (hariharanweb) [inline src/components/TravelClass.js:12]: Use grid instead of Div

**Review by hariharanweb — CHANGES_REQUESTED** [review]

File: src/components/TravelClass.js (hunk 1 above)
Comment (prochetab1999) [reply src/components/TravelClass.js:12]: Price component cannot be used as it seemed to me and Nidhi that the font-size is a little bit bigger and also the height and width attached to the Price component does not sit well with the TravelClass component

Comment (hariharanweb) [reply src/components/TravelClass.js:12]: Create variants. Like how typography does or icon sizes components  do

File: src/components/TravelClass.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <PriceTypography value={fareDetails?.price?.value} variant="caption" fontWeight={600} />
```
Comment (hariharanweb) [inline src/components/TravelClass.js:14]: You cannot use PriceTypography here; Again all the display related to pricing should be in <Price; Eg: Offer price, max_price etc.
think something like <Price variant="small" value={price}/>"
take inspiration from https://mui.com/material-ui/icons/#size

**Review by hariharanweb — CHANGES_REQUESTED** [review]

File: src/components/TravelClass.js (hunk 2 above)
Comment (prochetab1999) [reply src/components/TravelClass.js]: updated

**Pull request merged**